org.lflang/src/org/lflang/ASTUtils.java

```diff
@@ -1499,14 +1374,15 @@ public static boolean belongsTo(EObject eobject, Reactor reactor) {
      *  respective parameter or if the chain of instantiations is not nested.
      */
     public static Integer initialValueInt(Parameter parameter, List<Instantiation> instantiations) {
-        List<Value> values = initialValue(parameter, instantiations);
+        List<Expression> expressions = initialValue(parameter, instantiations);
         int result = 0;
-        for (Value value: values) {
-            if (value.getLiteral() == null) { 
+        for (Expression expr: expressions) {
+            if (!(expr instanceof Literal)) {
                 return null;
             }
             try {
-                result += Integer.decode(value.getLiteral());
+                // FIXME: why does this sum the values in the list??
```

Comment:
:thinking:

Reply:
This method is a little weird, but at least the Javadoc states that this is the desired behavior. I deleted the FIXME.